src/contracts/Metadata.contract.js

```diff
@@ -172,18 +160,24 @@ export default class Metadata {
     return pendingChanges
   }
 
-  async confirmPendingChange({ miningKeyToConfirm, senderVotingKey }) {
-    let alreadyConfirmed = await this.metadataInstance.methods
-      .isAddressAlreadyVoted(miningKeyToConfirm, senderVotingKey)
-      .call()
+  async confirmPendingChange({ miningKeyToConfirm, senderVotingKey, senderMiningKey }) {
+    let alreadyConfirmed
+    if (this.metadataInstance.methods.isValidatorAlreadyVoted) {
```

Comment:
Let's define `const { methods } = this.metadataInstance` in a line before and reuse it.

Reply:
@vbaranov Done in https://github.com/poanetwork/poa-dapps-validators/pull/73/commits/c7863415fb6901d070b7701217dc5c381e3ad8a1.